ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -186,12 +186,17 @@ impl VarLocation {
             },
             yksmp::Location::Constant(v) => {
                 let bitw = m.inst(iidx).def_bitw(m);
-                assert!(bitw <= 64);
+                assert!(bitw <= 32);
                 VarLocation::ConstInt {
                     bits: bitw,
                     v: u64::from(*v),
                 }
             }
+            yksmp::Location::LargeConstant(v) => {
```

Comment:
Are these changes relevant to this PR?

Reply:
Yes and no: the relevant changes to yklua trigger new traces that encounter these cases.

Reply:
Ah, that's fair!